spec/jobs/reactions/create_job_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
+
+RSpec.describe Reactions::CreateJob, type: :job do
+  include_examples "#enqueues_job", "reaction_create", [{ user_id: 8, reactable_id: 1, reactable_type: "Comment", category: "like" }]
+
+  describe "#perform_now" do
+    let(:reactable_type) { "Comment" }
+
+    context "when no user found" do
+      before do
+        allow(User).to receive(:find_by)
+        allow(Reaction).to receive(:create)
+      end
+
+      it "does not create a reaction" do
+        described_class.perform_now(user_id: 8, reactable_id: 1, reactable_type: "Comment", category: "like")
+        expect(Reaction).not_to have_received(:create)
+      end
+    end
+
+    context "when no reactable found" do
+      before do
+        allow(reactable_type.constantize).to receive(:find_by)
+        allow(Reaction).to receive(:create)
+      end
+
+      it "does not create a reaction" do
+        described_class.perform_now(user_id: 8, reactable_id: 1, reactable_type: "Comment", category: "like")
+        expect(Reaction).not_to have_received(:create)
+      end
+    end
+
+    context "when user + reactable" do
```

Comment:
Stubbing is a fine technique, but sometimes we can miss an error by using too much of it. I would add one test checking that the reaction is really created.